content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,301 @@
+- `aria-expanded` - A Boolean value meant to communicate when a dropdown is expanded
+- `aria-valuemin` - The minimum allowed value in a numerical input
+- `aria-valuemax` - The maximum allowed value of a numerical input
+
+Additional to `aria` props, [the `role` property](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#roles) acts to have the browser see and read an element as a different one. Again, this is a highly advanced (and often incorrectly deployed) API for complex apps. To learn more, [read through Mozilla's ARIA basics article.](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics)
+
+# Contrast is Cool {#contrast}
+
+While screen-readers are imperative to consider for frontend work, a site's visuals can help provide a good experience for many users. While a certain color palette may be aesthetically pleasing, it may be difficult to read for a colorblind user. Colorblind users aren't the only ones impacted, however.
+
+While there are various reasons a user might not be able to see weakly contrasted color, everyone is different. See if you can distinguish the text from the background in the displayed image with poor contrast:
+
+<img alt="Dark gray text on a black background" src="./color_fail.png" style="max-width: 600px; width: 100%"/>
+
+Now, compare that to highly contrasting colors: 
+
+<img alt="White text on a black background" src="./color_pass.png" style="max-width: 600px; width: 100%"/>
+
+The contents are not only easier to distinguish from the background, but it's made easier to focus on as a result of the increased contrast.
+
+This said, not all contrasts are the same. Per [WCAG guidelines](#wcag), you may have a different ratio of contrast for different compliance levels. These contrast ratios depend on both font size as well as compliance level.
+
+![Light grey text passing AA for large text, but not small text](./pass_aa_large.png)
+
+In this example you can see that the text passes the WCAG AA requirements for large text, but fails the same requirements for small text.
+
+# Fantastic Fonts {#font-resize}
+
+One of the most widely used accessibility features is font scaling. While many browsers default to a font size of `16px`, the user is actually able to change settings on their device to configure websites to use a larger font size.
+
+Many phones using iOS and Android allow users to change the font size on their mobile devices. This feature is heyso commonplace that it often prompts users to change this setting when the phone is being set up.
+
+<div style="display: flex; justify-content: space-around">
+    <figure>
+		<img src="./ios_text_size.png" style="height: 300px" alt=""/>
+        <figcaption>
+            iOS font size settings screen
+        </figcaption>
+    </figure>
+    <figure>
+        <img src="./android_text_size.png" style="height: 300px" alt=""/>
```

Comment:
My SR reads out the src attribute here. This is only a UU issue. Is Gatsby messing with the alt?

Reply:
Oh dear, it certainly is:

```
<img class="gatsby-resp-image-image medium-zoom-image" alt="android text size" title="android text size" src="/static/4a59c35d2d537f47a62e561e3c49f22a/fcda8/android_text_size.png">
```

I'll take a look at the `gatsby-medium-zoom` plugin. We've had to fork in the past, so I'm sure we'll be able to get this fixed. Thanks a million for the callout.

This might delay how quickly we can deploy this article, but we've got to wait for it to be published on CoderPad anyway so I'll try to sync those releases